### DISCUSSION TITLE
No such file or directory: 'matariki-tech-logo.png'

### DISCUSSION THREAD

**Discussion by Qmeriaux** [csgoh/roadmapper#43] [Q&A] [answered]
I just installed Python 3.11.4 and pip installed roadmapper (so theses are fresh installs).
I copy pasted the default code available in the github, and got this error : 
![image](https://github.com/csgoh/roadmapper/assets/35970288/6d33ec88-a1d9-4ef7-ad6f-cb181e6910ad)

**Comment by felix-seifert** [comment, marked as answer]
As the last error line indicates, the example code you used wants to open the file `matariki-tech-logo.png` for implementing a custom logo for the roadmap. You should ensure that this file is present at the required location or remove the requirement of this file.

**Comment by csgoh** [comment]
@Qmeriaux Download the logo from https://github.com/csgoh/roadmapper/blob/main/images/logo/matariki-tech-logo.png into your local and run your code again. That will fix your issue.
